readme-template.md

```diff
@@ -0,0 +1,54 @@
+# [BOX NAME]
+
+- [Requirements](#requirements)
+- [Setup](#setup)
+    - [Installation](#installation)
+    - [Deployment](#deployment)
+- [Usage](#usage)
+    - [Commands](#commands)
+    - [Testing](#testing)
+- [Support](#support)
+
+[BOX DESCRIPTION: Here you should introduce what your box is, give any background information if necessary, and why people should use it!]
+
+## Requirements
+
+The [BOX NAME] box has the following requirements:
+
+- [REPLACE OR ADD REQUIREMENTS AND HOW TO DOWNLOAD]
+- [Node.js](https://nodejs.org/) 10.x or later
+- [NPM](https://docs.npmjs.com/cli/) version 5.2 or later
+- 
```

Comment:
Look like this might be a spurious bullet point

Reply:
I left it there so that they can add more requirements, but I can remove it